### ISSUE TITLE
[bug] cpu_count not propagated to boost b2

### ISSUE THREAD

**Issue by jputcu** [conan-io/conan-center-index#13181]

**Comment by jwillikers** [comment]
Yes, the Boost recipe really needs to be updated to use the B2 generator. A workaround might be possible by passing a flag to B2 via the `extra_b2_flags` option for the Boost package. Try using `-o 'boost/*:extra_b2_flags=-j 1''. That appears to work for me. Let me know if you run into any other problems.
